src/config.ts

```diff
@@ -20,10 +20,32 @@ export const config = {
         password: process.env.RMQ_PASSWORD || 'guest',
         username: process.env.RMQ_USERNAME || 'guest',
     },
+    validator: {
+        user: {
+            id: /^[\w-]+@[\w-]+$/,
```

Comment:
In my opinion the standard for user unique id is `^\w+@\w+$\` no need for `-`.
Prove me wrong and I'll accept this.

Reply:
I proved you!!